lib/display.go

```diff
@@ -207,6 +207,11 @@ func highlightAlgorithm(sigAlg simpleSigAlg) string {
 	return color.SprintFunc()(algString(sig))
 }
 
+// timeString formats a time in UTC with minute precision, in the given color.
+func timeString(t time.Time, c *color.Color) string {
+	return c.SprintfFunc()(t.Format("2006-01-02 15:04Z"))
```

Comment:
Oh yeah, I wanted Z0700.  Is "Valid: 2016-06-10 22:14 UTC to 2023-04-15 22:14 UTC" good, or should I do t.UTC().Format, and hard-code a "UTC" in the `Valid: %s to %s (UTC)`